OpenSim/OpenSim.h

```diff
@@ -25,6 +25,8 @@
 
 #include "Common/osimCommon.h"
 #include "Simulation/osimSimulation.h"
+#include "Simulation/SimulationUtilities.h"
```

Comment:
I think this include should be in `Simulation/osimSimulation.h`, *not* this file.